std/testing/asserts.ts

```diff
@@ -56,12 +56,10 @@ function buildMessage(diffResult: ReadonlyArray<DiffResult<string>>): string[] {
   );
   messages.push("");
   messages.push("");
-  diffResult.forEach(
-    (result: DiffResult<string>): void => {
-      const c = createColor(result.type);
-      messages.push(c(`${createSign(result.type)}${result.value}`));
-    }
-  );
+  diffResult.forEach((result: DiffResult<string>): void => {
```

Comment:
Strangely, this unrelated block seems to have been changed when formatting this module. Shall I revert this? I appreciate it's a bit misleading given it isn't directly related to my changes.

Reply:
Disregard. This was a symptom of my using `deno fmt` directly *slaps on wrist*.